.github/workflows/build-images-ci.yaml

```
@@ -244,6 +244,30 @@ jobs:
             NOSTRIP=1
             OPERATOR_VARIANT=${{ matrix.name }}
 
+      - name: Install Bom and generate SBOM
+        shell: bash
+        run: | 
+          curl -L https://github.com/kubernetes-sigs/bom/releases/download/v0.3.0/bom-linux-amd64.tar.gz -o bom 
```

[Nit 1/3] Since `bom` releases provide signatures via `.sig` files, and we're dealing with signing stuff, how about we check the archive as well...
```suggestion
          curl -sSL --remote-name-all https://github.com/kubernetes-sigs/bom/releases/download/v0.3.0/bom-linux-amd64.tar.gz{,.sig}
          sha256sum --check bom-linux-amd64.tar.gz
```

---

I ran the commands locally to test the changes, running `sha256sum --check bom-linux-amd64.tar.gz` throws error: `sha256sum: bom-linux-amd64.tar.gz: no properly formatted SHA256 checksum lines found`. As per #sig-release of K8s, .sig is not sha256sum files, but something else: 
https://app.slack.com/client/T09NY5SBT/C2C40FMNF/thread/C2C40FMNF-1664035919.690419

---

Ah yes, looking at the file it's probably a cosign signature or something. Let's not bother with that then:
```suggestion
          curl -sSL --remote-name-all https://github.com/kubernetes-sigs/bom/releases/download/v0.3.0/bom-linux-amd64.tar.gz
```